force-app/main/default/lwc/errorSettings/errorSettings.js

```diff
@@ -41,6 +70,18 @@ export default class ErrorSettings extends LightningElement {
         return undefined;
     }
 
+    get showRequired() {
+        return !this.affordancesDisabled;
```

Comment:
minor here but the logic here is a bit odd imo. Are we saying if affordance enabled then it is required? I know we are only the boolean value here but maybe easier if just passing the values themselves?

Reply:
This is to display the required red star, so it's only displaying it if affordances are enabled.

Reply:
Changed to showRequiredIndicator and added a comment